cmd/earthly/subcmd/build_cmd.go

```diff
@@ -807,6 +754,112 @@ func receiveFileVersion2(ctx context.Context, conn io.ReadWriteCloser, localArti
 	return f.Close()
 }
 
+func (a *Build) runnerName(ctx context.Context) (string, bool, error) {
```

Comment:
I split a few sections of code out into their own methods.

Reply:
nice refactor!

I personally find `runnerName` to be a bit confusing (I had to look it up under the original code to see what it does), maybe we should rename it while we're refactoring this.

I'm not entirely sure what it's used for, but it appears to be some sort of short-hand name of the buildkit instance we're connecting to, which gets passed to logbus, and sent in a proto field that's called "Runner".

maybe we should call it `buildkitInstanceName` instead? or `logBusRunnerName`?

runner seems too vague and could get confused with our the "runner" process we used from our CI intrastructure that wraps the `earthly` binary.

Reply:
The lingo is used in a few other places. I'm not sure that we should change it everywhere.

https://github.com/earthly/earthly/blob/mh%2Fauto-skip-arg-passing/earthfile2llb/earthfile2llb.go#L174
https://github.com/earthly/earthly/blob/mh%2Fauto-skip-arg-passing/earthfile2llb/converter.go#L2393

Reply:
feel free to ignore it, I think a simple comment for the method would be good enough for now.

Reply:
maybe something like:
```
// runnerName returns the name of the logbus runner; which is a representation of what buildkit instance is being used, e.g. local:<hostname>, sat:<org>/<name>, or bk:<remote-address>
```

Reply:
SGTM!